.github/workflows/codeql.yml

```diff
@@ -91,11 +91,17 @@ jobs:
             libzstd-dev \
             unixodbc-dev
 
+    - name: Install latest ninja
+      run: pip install ninja
```

Comment:
better install ninja through the apt package ``ninja-build``

Reply:
I'll need to redo benchmarks if we choose that old of a version, especially since the ordering was improved in 1.12.0, that might be the only reason why it was a bit faster than make, rendering my assumptions now false.

Reply:
ok, let's keep pip then

Reply:
You could pin to a specific version if you'd like, as opposed to the system's package manager.